lib/index.js

```diff
@@ -143,6 +147,36 @@ function change (data, callback) {
 }
 exports.change = change;
 
+function checkForScopedDependencies(data, callback) {
+    if (!data.json['dist-tags']) {
```

Comment:
I can imagine a situation where package A has a dependency on @foo/B where @foo/B doesn't have any dist-tags set. This is probably (hopefully) super-corner case though.